cms/envs/devstack.py

```diff
@@ -300,6 +300,7 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 EVENT_BUS_REDIS_CONNECTION_URL = 'redis://:password@edx.devstack.redis:6379/'
 EVENT_BUS_TOPIC_PREFIX = 'dev'
 EVENT_BUS_CONSUMER = 'edx_event_bus_redis.RedisEventConsumer'
+EVENT_BUS_XBLOCK_LIFECYLCE_TOPIC = 'course-authoring-xblock-lifecycle'
```

Comment:
shouldn't this be `EVENT_BUS_TOPIC_PREFIX+'course-authoring-xblock-lifecycle'`

Reply:
@sameenfatima78 The topic prefix is combined in the underlying implementation. [Redis implementation](https://github.com/openedx/event-bus-redis/blob/c636eca2b36666aa22daa1349f0a4636d3fcee5e/edx_event_bus_redis/internal/config.py#L36-L51) and [Kafka implementation](https://github.com/openedx/event-bus-kafka/blob/1f3896a9d6a2f45dd2b07e576a910a9161d236f3/edx_event_bus_kafka/internal/config.py#L116-L131)

Reply:
@navinkarkera Why is this added only in the devstack settings? Wouldn't it make more sense to add it in base settings and override it in devstack if needed?

Reply:
I had the same thought but gave up and decided to add the setting close to the feature toggle which is also temporary and will go away with the implementation of ADR mentioned in description. :smiley_cat: